staking/availability/measure.go

```diff
@@ -16,8 +16,12 @@ import (
 
 var (
 	measure                    = new(big.Int).Div(big.NewInt(2), big.NewInt(3))
-	errValidatorEpochDeviation = errors.New("validator snapshot epoch not exactly one epoch behind")
-	errNegativeSign            = errors.New("impossible period of signing")
+	errValidatorEpochDeviation = errors.New(
+		"validator snapshot epoch not exactly one epoch behind",
+	)
+	errNegativeSign = errors.New("impossible period of signing")
+	// ErrDivByZero ..
+	ErrDivByZero = errors.New("toSign of availability cannot be 0, mistake in protocol")
```

Comment:
"mistake in protocol" => "This shouldn't happen"; this is no mistake in protocol :)